entrypoint.sh

```diff
@@ -20,15 +22,16 @@ fi
 git config --global user.name "$GIT_USERNAME"
 git config --global user.email "$GIT_EMAIL"
 
-test -d /backup/git/ || git clone --depth 1 $GIT_REPO /backup/git --branch $GIT_BRANCH || git clone $GIT_REPO /backup/git
-cd /backup/git/
-git checkout ${GIT_BRANCH} || git checkout -b ${GIT_BRANCH}
+test -d "$GIT_REPO_PATH" || git clone --depth 1 "$GIT_REPO" "$GIT_REPO_PATH" --branch "$GIT_BRANCH" || git clone "$GIT_REPO" "$GIT_REPO_PATH"
+cd "$GIT_REPO_PATH"
+git checkout "${GIT_BRANCH}" || git checkout -b "${GIT_BRANCH}"
 git rm -r . || true
```

Comment:
This part hasn't changed - only "/backup/git" was moved to a variable with name `GIT_REPO_PATH` to not hardcode anything.